lib/tasks/achievements_import.rake

```diff
@@ -0,0 +1,14 @@
+namespace :desperado do
+
+  desc 'Import achievements into the DB.'
+  task :achievements_import, [:data_file] => :environment do |_, args|
+    File.open("#{Rails.root}/#{args.data_file}", 'r') do |data|
+      achievements = Psych.load_stream data
+      achievements.each do |achievement|
+        Achievement.create achievement
+      end
```

Comment:
[It does!](http://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-create) Johno, you are a :star: thanks!

I have updated the code to reflect that. I also wrapped the `create` call in a single transaction so that either all the acheevs are saved or none of them are on a rollback.